### PR TITLE
Update Nix version to 2.10.3 on CI

### PR DESCRIPTION
**Pull request by adrianotm**

### CODE REVIEW

**Review by FPtje — CHANGES_REQUESTED** [review]
Thanks!

Could you update line 4 of `nix/install`?  It currently has an out-of-date retrieval date.

File: nix/install [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # Vendored Nix installation script.
-# Retrieved 2020-01-08 from https://nixos.org/nix/install
+# Retrieved 2022-09-22 from https://nixos.org-2.10.3/nix/install
```
Comment (FPtje) [inline nix/install:4]: Nit: This URL is invalid, I think the `-2.10.3` can be removed.

**Review by FPtje — APPROVED** [review]
LGTM besides one nit

File: nix/install (hunk 1 above)
Comment (adrianotm) [reply nix/install:4]: `2.10.3` is not the latest version, so `https://nixos.org/nix/install` redirects to `2.11.1` (currently).

**Comment by adrianotm** [comment]
@OpsBotPrime merge

**Comment by OpsBotPrime** [comment]
Pull request approved for merge by @adrianotm, rebasing now.

**Comment by OpsBotPrime** [comment]
Rebased as d1082ae017231a47a14ddfff363b7d8d4f8272a0, waiting for CI …

**Comment by OpsBotPrime** [comment]
[CI job :yellow_circle:](https://channable.semaphoreci.com/workflows/4cc7433b-21f1-4128-afef-75b543117c18?pipeline_id=1636511f-98d3-408a-b315-b5f7d79e9044) started.

**Pull request merged**